dongkyu/initial-settings-express/myapp/db/migrations/20221220040036_create_books_authors_table.sql

```diff
@@ -0,0 +1,11 @@
+-- migrate:up
+CREATE TABLE books_authors (
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  book_id INT NOT NULL,
+  author_id INT NOT NULL,
+  CONSTRAINT book_authors_book_id_fkey FOREIGN KEY (book_id) REFERENCES books(id),
+  CONSTRAINT book_authors_author_id_fkey FOREIGN KEY (author_id) REFERENCES authors(id)
```

Comment:
제약 조건을 잘 지정해주셨네요! 👍

마이너한 수정 사항으로 book_authors_... 대신 books_authors_...로 수정해주면 좋을 것 같습니다.

```suggestion
 CONSTRAINT books_authors_book_id_fkey FOREIGN KEY (book_id) REFERENCES books(id),
 CONSTRAINT books_authors_author_id_fkey FOREIGN KEY (author_id) REFERENCES authors(id)
```

Reply:
해당 부분도 수정하셔서 다시 push 해주세요~!